source/text.cpp

```diff
@@ -622,7 +623,7 @@ spv_result_t spvTextEncodeOpcode(const spvtools::AssemblyGrammar& grammar,
           break;
         } else {
           return context->diagnostic()
-                 << "Expected operand, found end of stream.";
+                 << opcodeName << " Expected operand, found end of stream.";
```

Comment:
It would be better if the opcode name formed part of the sentence.  The same goes for most of the other changes.  For example:

```
Expected operand for OpTypeForwardPointer instruction, but found the end of the stream.
```

instead of

```
OpTypeForwardPointer Expected operand, found end of stream.
```

Reply:
agree, let me refactor the error messages